documentation/smg/10_build_release_process.md

```diff
@@ -1,85 +1,156 @@
+- The Herbert `master` branch is built and a zip release build artifact with
+MATLAB-release and platform-specific binary is created.
 
-- The Horace `master` branch is built and a zip release build artifact with MATLAB-release and platform-specific binary is created, which includes the last successful Herbert `master` build.
+- The Horace `master` branch is built and a zip release build artifact with
+MATLAB-release and platform-specific binary is created, which includes the last
+successful Herbert `master` build.
 
-The Horace build artifact is a single deployable unit to a specific MATLAB/OS platform. This is equivalent to the artifact created by the legacy MATLAB create release process.
+The Horace build artifact is a single deployable unit to a specific MATLAB/OS
+platform.
+This is equivalent to the artifact created by the legacy MATLAB create release
+process.
 
-### Features
+## Features
 
-Feature development and bug fixes should be performed on branches and reviewed using a GitHub Pull Request (PR).
+Feature development and bug fixes should be performed on branches and reviewed
+using a GitHub Pull Request (PR).
 
-When a pull request is opened or updated the merged code of the PR branch and `master` is built and tested.
+When a pull request is opened or updated the merged code of the PR branch and
+`master` is built and tested.
 
 1. Feature developed on a branch taken from the `master`
-2. All tests likely to be affected by changes in Herbert and Horace must be run on developers machine
+2. All tests likely to be affected by changes in Herbert and Horace must be run
+ on developers machine
 3. Create Herbert PR (if required) for merge into `master` branch
-4. Create Horace PR (if required) - if a Herbert PR has been created too the Horace PR must not be merged until the Herbert PR has been merged
-5. If the build / test of the PR against the Release branch succeeds on all platforms, and the code has been reviewed, the PR can be merged.
+4. Create Horace PR (if required) - if a Herbert PR has been created too the
+Horace PR must not be merged until the Herbert PR has been merged
+5. If the build/test of the PR against the Release branch succeeds on all
+platforms, and the code has been reviewed, the PR can be merged.
 
-### Releases
+## Releases
 
-The Jenkins build artifacts are are not accessible outside of STFC. End users will be directed to GitHub to access packaged releases.
+The Jenkins build artifacts are are not accessible outside of STFC.
+End users will be directed to GitHub to access packaged releases.
 
-A 'deploy' pipeline will be created ([Horace #73](https://github.com/pace-neutrons/Horace/issues/73)).
+A 'deploy' pipeline will be created
+([Horace #73](https://github.com/pace-neutrons/Horace/issues/73)).
 
 This will:
 
-- tag the `master` branch with the release number (`Rm_n_o`)
+- tag the `master` branch with the release number (`Rm.n.o`)
```

Comment:
The tag we have on Horace at the moment is `REL_3_4_1_beta` and the patch release section below still uses `_`

Reply:
Ignore this. You are correct, this is the RELEASE number not the RELEASE_TAG.